CHANGELOG.md

```diff
@@ -46,7 +46,7 @@ A big thanks to the 13 contributors who made this release possible. Here are som
 - &#8203;<!-- 29 -->[docs] Improve docs for creating dark theme (#28104) @mnajdova
 - &#8203;<!-- 28 -->[docs] Don't use Material theme in unstyled demos (#28073) @michaldudak
 - &#8203;<!-- 27 -->[docs] Fix api doc import example (#28199) @siriwatknp
-- &#8203;<!-- 26 -->[docs] Remove demo for re-creating Material-UI switches (#28042) @eps1lon
+- &#8203;<!-- 26 -->[docs] Remove demo for re-creating MUI switches (#28042) @eps1lon
```

Comment:
It's unclear to me if we should update the changelog. One could argue that it's almost immutable.

Reply:
Yeah, agree.

Reply:
Will revert the changelog changes